code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -209,8 +209,12 @@
 /datum/reagent/water/holywater/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
 	..()
 	if(ishuman(M)) // Any location
-		if(M.mind && cult.is_antagonist(M.mind) && prob(10))
-			cult.remove_antagonist(M.mind)
+		if(iscultist(M))
+			if(prob(10))
+				cult.offer_uncult(M)
+			if(prob(2))
+				var/obj/effect/spider/spiderling/S = new /obj/effect/spider/spiderling(M.loc)
```

Comment:
So...you can be forced into the cult, but you can only voluntarily leave? That's really stacking the deck against the crew.

Also, the ability to make spiderlings means that cultists can cram tons of holy water down their throats to spam the station with spiders--and since it's voluntary, well, they never have to worry about negative side effects.

code wise? get_turf(M) not m.loc